examples/object-initializer/object_initializer.description

```diff
@@ -1,3 +1 @@
-// In Ballerina, objects can have a constructor that can be used to initialize the object.
-// This constructor can have parameters similar to function parameters, and you can refer to
-// object fields directly as constructor parameters.
+// In Ballerina, objects can have a constructor that can be used to initialize the object.
```

Comment:
Can we expand on this description and add some details for more comprehensive understanding?